src/types.ts

```diff
@@ -220,19 +220,6 @@ export interface RunnerOptionConfig {
   verbose?: boolean
 }
 
-export interface ConnectionConfig {
```

Comment:
**suggestion:** Maybe someone also rely on `ConnectionConfig`
Cause we exported this here, you should also export it from `pg`